### PR TITLE
School save

### PR DESCRIPTION
**Pull request by adelaidemz**
- Save updated lists (schools and/or list settings) to backend
- Functionality for removing school from list

### CODE REVIEW

File: src/frontend/shortlist/src/views/CategorizeView.vue [hunk 1]
```diff
@@ -196,12 +199,25 @@ export default {
...
+      this.saveList(e.listId);
     },
     shareList(e) {
       console.log("SHARED LIST #:", e);
     },
+    deleteFromList() {
+      // console.log("DELETE FROM LIST: ", this.schoolDetailModalData.data.schoolMetadata.id);
+      let schoolIdx = this.schoolDetailModalData.schoolIdx;
+      let listIdx = this.schoolDetailModalData.listIdx;
+      this.myShortlists[listIdx].school_ids.splice(schoolIdx, 1);
```
Comment (simaoalicechen) [inline src/frontend/shortlist/src/views/CategorizeView.vue:213]: I don't remember myShortlists[listIdx] was. Is this line supposed to mark the school as neither trashed or accepted?

**Review by simaoalicechen — COMMENTED** [review]
I just have one question, if you can clear that, I think this is good.

File: src/frontend/shortlist/src/views/CategorizeView.vue (hunk 1 above)
Comment (adelaidemz) [reply src/frontend/shortlist/src/views/CategorizeView.vue:213]: This just removes the school from the list, separate from marking it trashed or accepted, but I will add a call to mark as rejected

Comment (adelaidemz) [reply src/frontend/shortlist/src/views/CategorizeView.vue:213]: This just removes the school from the list, separate from marking it trashed or accepted, but I will add a call to mark as rejected

**Pull request merged**